public/stylesheets/style.css

```diff
@@ -21,3 +21,8 @@
   text-decoration: line-through;
   color: #777;
 }
+
+label.alert-danger {
```

Comment:
I think this can be handled without overriding bootstrap styles - in the bootstrap form documentation, it looks like we need to provide a `has-error` class to the form group container?
```html
<div class="has-error">
```